sdk/core/core-lro/src/pollOperation.ts

```diff
@@ -0,0 +1,12 @@
+export interface PollOperationState<T> {
+  completed?: boolean;
```

Comment:
do we need a boolean flag for 'started' to indicate the initial operation has begun?